### DISCUSSION TITLE
How to execute GHA locally with MacOsx as runner

### DISCUSSION THREAD

**Discussion by malik-coachhub** [nektos/act#1450] [Q&A] [answered]
am working to create a CI CD process to upload our IoS build to Testlight using Github Actions. After setting the Fastlane file, and having the .yml file for Github actions. I want test the Github Actions locally to build IoS. I tried to use [act](https://github.com/nektos/act) but it doesn't include a Macos runner. What would be the best approach to run the Github Actions Locally for IoS build? Is there any other tools or some work around?

Be able to build IOS build using Github action Locally

**Comment by ChristopherHX** [comment, marked as answer]
You can run without containers starting with act v0.2.34 scheduled for release on 1. Dec. 2022.

until then [grab a snaphot of act](https://github.com/nektos/act/actions/runs/3525873379) or wait for the release.

e.g.
`act -P macos-latest=-self-hosted`

`-self-hosted` means run on host, no container isolation.
